src/util/CookieUtil.js

```diff
@@ -54,5 +55,26 @@ define(['okta', 'vendor/plugins/jquery.cookie'], function (Okta) {
     removeCookie(REMEMBER_DEVICE_COOKIE_NAME);
   };
 
+  fn.isAutoPushEnabled = function (username) {
```

Comment:
Instead of username, can you use the "userId" ? 
The path to that attribute in the response JSON is: "_embedded->user->id".